examples/export_figure.py

```diff
@@ -2,9 +2,11 @@
 Export Figure
 =============
 
-Display one shapes layer ontop of one image layer using the ``add_shapes`` and
-``add_image`` APIs. When the window is closed it will print the coordinates of
-your shapes.
+Display a variety of layer types in the napari viewer and export the figure with `viewer.export_figure()`.
+The exported figure is then added back as an image layer.
+
+Exported figures include the extent of all data in 2D view, and does not presently work for 3D views.
+To capture the extent of the canvas, instead of the layers, see `viewer.screenshot()`: `examples/to_screenshot.py` and `examples/screenshot_and_export_figure.py`.
```

Comment:
Tiny nit: do we want to link to the gallery version of these examples instead?

Reply:
Oh that would make sense since this doctoring is primarily for the gallery!

Reply:
@melissawm how would such links work? Do you mean hardcoding the URL? What is the default sphinx anchor for gallery examples?

Reply:
Would love this, but not sure how. I would definitely want some kind of cross reference, like I'm able to do in mkdocs `[display name](examples/example.py)`
[It looks like sphinx has something](https://docs.readthedocs.io/en/stable/guides/cross-referencing-with-sphinx.html#the-doc-role), but I can't figure out how to get it to work in my local docs build (maybe because it crosses to napari/napari to build the gallery). Should be something like `:doc:`/examples/to_screenshot`

Reply:
I peeked at https://napari.org/stable/gallery/surface_multi_texture.html#sphx-glr-gallery-surface-multi-texture-py and made suggestions to try, see below.